rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  belongs_to :student
+  belongs_to :book
+
+  attr_accessor :date
```

Comment:
- [ ] Make sure Getters and Setters for @book and @person instance variables should be present.